Gemfile.lock

```diff
@@ -207,7 +208,7 @@ GEM
     fastlane-plugin-appcenter (2.1.2)
     fastlane-plugin-sentry (1.24.0)
       os (~> 1.1, >= 1.1.4)
-    fastlane-plugin-wpmreleasetoolkit (12.0.0)
+    fastlane-plugin-wpmreleasetoolkit (12.1.0)
```

Comment:
This is unrelated to the fix. It was just handy to upgrade to:

1. Make this available for the next code freeze
2. Not get prompted every time I run the tests locally